pkg/controller/operators/olm/operator.go

```diff
@@ -1246,7 +1246,7 @@ func (a *Operator) allNamespaceOperatorGroups() ([]*v1.OperatorGroup, error) {
 }
 
 func (a *Operator) syncOLMConfig(obj interface{}) (syncError error) {
-	a.logger.Debug("Processing olmConfig")
+	a.logger.Debugf("Processing olmConfig")
```

Comment:
Switching from `Debug` to `Debugf` shouldn't change the level at which it's output.

Reply:
Yes, that idea didn't really make sense. Still trying to track down why this log comes through though.